packages/wp-dashboard/src/api/authors.js

```diff
@@ -27,7 +27,7 @@ export function getAuthors(config, search) {
   return apiFetch({
     path: addQueryArgs(config.api.users, {
       per_page: '100',
-      who: 'authors',
+      capabilities: 'edit_web-stories',
```

Comment:
Wonder if this should come from config.

Reply:
Makes sense I'll let you pick this up as I won't get back to it until tonight.